rocAL/rocAL/source/augmentations/geometry_augmentations/node_warp_affine.cpp

```diff
@@ -60,23 +60,23 @@ void WarpAffineNode::create_node() {
 
     _node = vxExtRppWarpAffine(_graph->get(), _inputs[0]->handle(), _inputs[0]->get_roi_tensor(), _outputs[0]->handle(), _affine_array,
                                interpolation_vx, input_layout_vx, output_layout_vx, roi_type_vx);
-    if((status = vxGetStatus((vx_reference)_node)) != VX_SUCCESS)
+    if ((status = vxGetStatus((vx_reference)_node)) != VX_SUCCESS)
         THROW("Adding the warp affine (vxExtRppWarpAffine) node failed: " + TOSTR(status))
 }
 
 void WarpAffineNode::update_affine_array() {
-    for (uint i = 0; i < _batch_size; i++ ) {
-        _affine[i*6 + 0] = _x0.renew();
-        _affine[i*6 + 1] = _y0.renew();
-        _affine[i*6 + 2] = _x1.renew();
-        _affine[i*6 + 3] = _y1.renew();
-        _affine[i*6 + 4] = _o0.renew();
-        _affine[i*6 + 5] = _o1.renew();
+    for (uint i = 0; i < _batch_size; i++) {
+        _affine[i * 6 + 0] = _x0.renew();
+        _affine[i * 6 + 1] = _y0.renew();
+        _affine[i * 6 + 2] = _x1.renew();
+        _affine[i * 6 + 3] = _y1.renew();
+        _affine[i * 6 + 4] = _o0.renew();
+        _affine[i * 6 + 5] = _o1.renew();
     }
     vx_status affine_status;
-    affine_status = vxCopyArrayRange((vx_array)_affine_array, 0, _batch_size * 6, sizeof(vx_float32), _affine.data(), VX_WRITE_ONLY, VX_MEMORY_TYPE_HOST); //vxAddArrayItems(_width_array,_batch_size, _width, sizeof(vx_uint32));
-    if(affine_status != 0)
-        THROW(" vxCopyArrayRange failed in the WarpAffine(vxExtRppWarpAffinePD) node: "+ TOSTR(affine_status))
+    affine_status = vxCopyArrayRange((vx_array)_affine_array, 0, _batch_size * 6, sizeof(vx_float32), _affine.data(), VX_WRITE_ONLY, VX_MEMORY_TYPE_HOST);  // vxAddArrayItems(_width_array,_batch_size, _width, sizeof(vx_uint32));
+    if (affine_status != 0)
```

Comment:
@fiona-gladwin remove this comment if not required

Reply:
Done